feature/bgp/policybase/otg_tests/extcomm_action_test/README.md

```diff
@@ -0,0 +1,262 @@
+      * conditions/bgp-conditions/match-ext-community-set/config/ext-community-set =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='match_ext_comms']
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='add_ext_comms']
+      * actions/bgp-actions/set-ext-community/config/options = ADD
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * For each policy-definition created, run a subtest (RT-7.8.3.x-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify routes are received on ATE port 1 for accepted prefixes.
+    * Verify expected communities are present.
+
+    * Expected matches for each policy
+      |              | add_ext_comms                                               | match_and_add_ext_comms                                     |
+      | ------------ | ----------------------------------------------------------- | ----------------------------------------------------------- |
+      | prefix-set-1 | [ 100:100000, 200000:200000 ]                               | none                                                        |
+      | prefix-set-2 | [ 100:100000, 200000:200000, 500:500, 600:600 ]             | [ 500:500, 600:600 ]                                        |
+      | prefix-set-3 | [ 100:100000, 200000:200000, 500000:500000, 600000:600000 ] | [ 100:100000, 200000:200000, 500000:500000, 600000:600000 ] |
+
+* RT-7.8.4 - Validate community-sets and routing-policy using OC release 3.x
+  * Note, this is the same at RT-7.8.2, but with a change in the location of the
+    `match-set-options` leaf which moved to
+    `/routing-policy/policy-definitions/policy-definition/policy-definition/bgp-conditions/match-ext-community-set/config/match-set-options`
+
+* RT-7.8.5 - Validate ext-community-sets and routing-policy using OC release 3.x
+  * Note, this is the same at RT-7.8.3, but with a change in the location of the
+    `match-set-options` leaf which moved to
+    `/routing-policy/policy-definitions/policy-definition/policy-definition/bgp-conditions/match-ext-community-set/config/match-set-options`
+
+* RT-7.8.6 - Validate adding link-bandwidth ext-community-sets using OC model release 2.x
+  * Add prefixes with link-bandwidth community to ATE port 1 to advertise ipv4
+    and ipv6 prefixes to DUT port 1 using the following communities:
+    * prefix-set-5 with 2 routes with no communities
+    * prefix-set-6 with 2 routes with communities `[ 100:100 ]`
```

Comment:
This would require ADD-PATH, which is not requested AFAIK.
And I think it is NOT requited, unless you want to verify wECMP forwarding, not just BGP policy handling. I think wECMP verification shpould be separat test and not included here.
```suggestion
    * prefix-set-5 with no communities
    * prefix-set-6 with communities `[ 100:100 ]`
```

Reply:
This test is not  intended to test ECMP (one route to two neighbors).  This is intended to be two different prefixes and this is only because using a single prefix could miss coverage of list functionality.  (Testing a list of one vs. testing a list of two)